library/worldgen/biome/src/main/java/org/quiltmc/qsl/worldgen/biome/api/BiomeSelectors.java

```diff
@@ -69,25 +70,23 @@ public static Predicate<BiomeSelectionContext> vanilla() {
 	 * assuming Vanilla's default biome source is used.
```

Comment:
The Javadoc for foundInOverworld hasn't been updated; It should have the same "This method is compatible with this tag" comment as the other two